src/designer/elsa-workflows-studio/src/components/screens/workflow-instance-list/elsa-workflow-instance-list-screen/elsa-workflow-instance-list-screen.tsx

```diff
@@ -26,6 +26,11 @@ import Tunnel from "../../../../data/dashboard";
   shadow: false,
 })
 export class ElsaWorkflowInstanceListScreen {
+  static readonly DEFAULT_PAGE_SIZE = 15;
+  static readonly MIN_PAGE_SIZE = 5;
+  static readonly MAX_PAGE_SIZE = 100;
+  static readonly START_PAGE = 0;
```

Comment:
I admire that you put these readonly statics here, but do they really need to scream at me? 😅 
Maybe `DefaultPageSize` etc. is just as good?

Reply:
Apologies on this, switching too many project and languages combined with too many style guides.
So I was unintentionally screaming at you with style guide that mandates constants to be ALL UPPERCASE :) (so it IS immediately visible they don't change over program execution).
Feel free to convert it to any style you want :).

Reply:
Completely understand, no worries 😃